COMETwebapp.Tests/Viewer/PrimitivesTests.cs

```diff
@@ -76,6 +76,7 @@ public void SetUp()
             var session = new Mock<ISessionAnchor>();
             this.context.Services.AddSingleton(session.Object);
 
+            this.context.Services.AddTransient<ISceneProvider, Scene>();

```

Comment:
it is good practice to name the implemenation of an interface in the same way, I propose you rename `Scene` to `SceneProvider`